package.json

```diff
@@ -34,7 +34,7 @@
     "author": "Wednesday Solutions",
     "license": "MIT",
     "engines": {
-        "node": "14.17.x"
+        "node": "14.19.x"
```

Comment:
Don't change the node engine version keep it to only 14.17.x

Reply:
changed it back to normal and added support for the node in dockerfile to work for 14.17.x